charts/posthog/templates/secrets.yaml

```diff
@@ -17,16 +17,3 @@ data:
   {{- if not .Values.email.existingSecret }}
   smtp-password: {{ default "" .Values.email.password | b64enc | quote }}
   {{- end }}
-
```

Comment:
Do we need the `$previous` above? Can't we make it idempotent without the lookup?

Reply:
It's still used to carry across the posthog-secret between runs

Reply:
Is there another way to handle this?

Reply:
Maybe there are some scenarios where removing it can trigger a breaking change but if our users either provide credentials as k8s secrets or as a plaintext value in `values.yaml`, this lookup should not be necessary.

Reply:
Hmm yes, the update is the tricky bit! I'm going to leave as is for now if that's ok.

Reply:
Sure 👍